client/extensions/woocommerce/app/product-categories/index.js

```diff
@@ -93,6 +94,9 @@ class ProductCategories extends Component {
 					<a href={ getLink( '/store/products/:site/', site ) }>{ productsLabel }</a>,
 					<span>{ categoriesLabel }</span>,
 				] }>
+				<Button primary href={ getLink( '/store/products/category/:site/', site ) }>
+					{ translate( 'Add category' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **71** times:
`translate( 'Add Category' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-base-product-cat-form)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).